tests/components/QuestionStore.test.ts

```diff
@@ -0,0 +1,77 @@
+import { test, expect, vi } from "vitest";
+import { getQuestionsByTagAndLimit } from "@components/QuestionStore";
+
+const questions = [
```

Comment:
Include a question without tags here (so we test it keeps working when the property is not there).